app/backends/workflow-orchestration.ts

```diff
@@ -781,21 +894,20 @@ export async function updateWorkItem(req: HarmonyRequest, res: Response): Promis
     duration,
   };
   const workItemLogger = req.context.logger.child({ workItemId: update.workItemID });
-  if (typeof global.it === 'function') {
-    // tests break if we don't await this
-    await handleWorkItemUpdate(update, operation, workItemLogger);
+
+  // we use separate queues for small and large work item updates
+  let queueType = WorkItemUpdateQueueType.SMALL_ITEM_UPDATE;
+  if (results?.length > 1) {
```

Comment:
Not sure why when more than one results, we set the queue type to LARGE_ITEM_UPDATE.